mathoxide-lib/src/array.rs

```diff
@@ -22,11 +117,13 @@ mod test {
 
     #[test]
     fn it_works() {
-        let array = Array::<ThreadSafeStorage<u32>>::zeros(10);
+        let array = Array::<ThreadSafeStorage<u32>, views::SimpleView>::zeros(&[4, 5]);
+        println!("{}", array);
```

Comment:
A better test here would be to get the formatted string using something like 

```rust
let formatted = format!("{}", array);
```

and then running some assertions on the resulting string